### PR TITLE
zh: Update the open source contribution guide document

### PR DESCRIPTION
**Pull request by warriorBrian**
## Fixed :speech_balloon:
* Update the open source contribution guide document to synchronize with the English document in 2.4.X version

### CODE REVIEW

File: zh/guide/contribution-guide.md [hunk 1]
```diff
@@ -7,15 +7,101 @@ description: 欢迎为 Nuxt.js 做出开源贡献！
 
 ## 提交问题
 
-开源贡献的其中一种方式是为项目提交问题的详细报告，以方便我们不断地迭代优化项目。
-我们欢迎并感谢大家提交有良好书写习惯且能够清晰描述问题的报告！遇到问题的时候，麻烦先详细阅读 Nuxt.js 的文档，并在 https://github.com/nuxt/nuxt.js/issues 上查找是否已有相同或类似的问题。
+我们使用[CMTY](https://cmty.nuxtjs.org/)使贡献者和维护者更容易的提交问题的详细报告。
+
+请确保包含一个复制代码仓库或[CodeSandBox](https://template.nuxtjs.org/)，以便可以更好地再现错误，我们开始修复它的速度越快！
```
Comment (clarkdo) [inline zh/guide/contribution-guide.md]: 克隆仓库

File: zh/guide/contribution-guide.md [hunk 2]
```diff
@@ -7,15 +7,101 @@ description: 欢迎为 Nuxt.js 做出开源贡献！
 
 ## 提交问题
 
-开源贡献的其中一种方式是为项目提交问题的详细报告，以方便我们不断地迭代优化项目。
-我们欢迎并感谢大家提交有良好书写习惯且能够清晰描述问题的报告！遇到问题的时候，麻烦先详细阅读 Nuxt.js 的文档，并在 https://github.com/nuxt/nuxt.js/issues 上查找是否已有相同或类似的问题。
+我们使用[CMTY](https://cmty.nuxtjs.org/)使贡献者和维护者更容易的提交问题的详细报告。
```
Comment (clarkdo) [inline zh/guide/contribution-guide.md]: 读不太通呀

File: zh/guide/contribution-guide.md (hunk 1 above)
Comment (warriorBrian) [reply zh/guide/contribution-guide.md]: 😸

File: zh/guide/contribution-guide.md (hunk 2 above)
Comment (warriorBrian) [reply zh/guide/contribution-guide.md]: 已经解决 😸

**Pull request merged**